docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
-| `ta_buffer_coalesced_write_cycles` | Cycles | Number of coalesced buffer write cycles           |
-| `ta_addr_stalled_by_tc`            | Cycles | Number of cycles texture addressing unit address is stalled by TCP     |
-| `ta_data_stalled_by_tc`            | Cycles | Number of cycles texture addressing unit data is stalled by TCP        |
-| `ta_addr_stalled_by_td_cycles[∗]`  | Cycles | Number of cycles texture addressing unit address is stalled by TD      |
-| `ta_flat_wavefronts`               | Instr  | Number of flat wavefront instructions             |
-| `ta_flat_read_wavefronts`          | Instr  | Number of flat read wavefront instructions        |
-| `ta_flat_write_wavefronts`         | Instr  | Number of flat write wavefront instructions       |
-| `ta_flat_atomic_wavefronts`        | Instr  | Number of flat atomic wavefront instructions      |
+| Hardware counter | Unit   | Definition |
+| :----------------------| :------| -------------------------|
+| `TA_TA_BUSY[n]` | Cycles | TA busy cycles (value range for n: [0-15]) |
+| `TA_TOTAL_WAVEFRONTS[n]` | Instr | Number of wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_BUFFER_WAVEFRONTS[n]` | Instr | Number of buffer wavefronts processed by TA (value range for n: [0-15] |
+| `TA_BUFFER_READ_WAVEFRONTS[n]` | Instr | Number of buffer read wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_BUFFER_WRITE_WAVEFRONTS[n]` | Instr | Number of buffer write wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_BUFFER_ATOMIC_WAVEFRONTS[n]` | Instr | Number of buffer atomic wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_BUFFER_TOTAL_CYCLES[n]` | Cycles | Number of buffer cycles (including read and write) issued to TC (value range for n: [0-15]) |
+| `TA_BUFFER_COALESCED_READ_CYCLES[n]` | Cycles | Number of coalesced buffer read cycles issued to TC (value range for n: [0-15]) |
+| `TA_BUFFER_COALESCED_WRITE_CYCLES[n]` | Cycles | Number of coalesced buffer write cycles issued to TC (value range for n: [0-15]) |
+| `TA_ADDR_STALLED_BY_TC_CYCLES[n]` | Cycles | Number of cycles TA address path is stalled by TC (value range for n: [0-15]) |
+| `TA_DATA_STALLED_BY_TC_CYCLES[n]` | Cycles | Number of cycles TA data path is stalled by TC (value range for n: [0-15]) |
+| `TA_ADDR_STALLED_BY_TD_CYCLES[n]`  | Cycles | Number of cycles TA address path is stalled by TD (value range for n: [0-15]) |
+| `TA_FLAT_WAVEFRONTS[n]` | Instr  | Number of flat opcode wavefronts processed by TA (value range for n: [0-15] )|
+| `TA_FLAT_READ_WAVEFRONTS[n]` | Instr  | Number of flat opcode read wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_FLAT_WRITE_WAVEFRONTS[n]` | Instr  | Number of flat opcode write wavefronts processed by TA (value range for n: [0-15]) |
+| `TA_FLAT_ATOMIC_WAVEFRONTS[n]` | Instr  | Number of flat opcode atomic wavefronts processed by TA (value range for n: [0-15]) |
 
 #### Texture data unit
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading.